diff --git a/bin.cjs b/bin.cjs
--- a/bin.cjs
+++ b/bin.cjs
@@ -84109,7 +84109,8 @@
 	}),
 	objectType({
 		driver: literalType("pglite"),
-		url: stringType().min(1)
+		url: stringType().min(1),
+		extensions: anyType().optional()
 	})
 ]);
 const printConfigConnectionIssues$2 = (options) => {
@@ -233312,7 +233313,7 @@
 				const { PGlite, types } = await import("@electric-sql/pglite");
 				const { drizzle } = await import("drizzle-orm/pglite");
 				const { migrate } = await import("drizzle-orm/pglite/migrator");
-				const pglite = "client" in credentials ? credentials.client : new PGlite(normalisePGliteUrl(credentials.url));
+				const pglite = "client" in credentials ? credentials.client : new PGlite(normalisePGliteUrl(credentials.url), credentials.extensions ? { extensions: credentials.extensions } : void 0);
 				await pglite.waitReady;
 				const drzl = drizzle({ client: pglite });
 				const migrateFn = async (config) => {
